### ISSUE TITLE
Idealised ventricle

### ISSUE THREAD

**Issue by iBatistic** [philipcardiff/paper-newton-krylov#4]
I reviewed the results we need to compare, and it seems extracting the data from the diagram using `WebPlotDigitizer` may be challenging. Since the repository containing these data is no longer accessible, I emailed the main author requesting the data. He replied and asked his former colleague to send the data to me.

**Comment by philipcardiff** [comment]
Great; thanks!

Yep, extracting the midline data from our model is also not so trivial.

**Comment by philipcardiff** [comment]
Comment on the choice of bulk modulus:

- The case definition specifies an incompressible material; however, they say that incompressibility can be enforced in any way we want.
- https://febio.org/knowledgebase/case-studies/structural-mechanics/cardiac-mechanics-benchmark-problems says *"a bulk modulus  also needs to be specified, which should be two to three orders of magnitude larger than the largest Young’s modulus."*
- The shear modulus is 1e4 Pa so the bulk modulus should be at least 1e6 Pa.
- In our approach (and many others), as we approach incompressibility, the convergence gets much slower.
- Setting the bulk modulus (k) equal to 1e6 Pa runs with a time step of 0.01 s and the results seem reasonable. Mesh 2 takes 857 s using k = 1e6 Pa. In contrast, the case is an order of magnitude faster when k = 1e5 Pa, but the results are 5-10% different.
- Let's go with k = 1e6 Pa, and we probably don't need to run very fine meshes (e.g. mesh3 may already be fine enough).

Next, I need to figure out an easy way to extract the mid-wall-line deformed position (maybe the authors will send you the VTK scripts for this).

**Comment by iBatistic** [comment]
I got the link to the results from the paper:

https://bitbucket.org/vgurev/mechbench/src/master/

**Comment by philipcardiff** [comment]
Excellent!

We can use this to generate our figures.

**Comment by philipcardiff** [comment]
> I got the link to the results from the paper:
> 
> https://bitbucket.org/vgurev/mechbench/src/master/

This repo does not seem to contain the graphs from the paper.

Instead, it contains the raw results from each group (e.g. deformed VTKs); however, the utilities to extract the data from the paper (e.g. deformed midlines) are not included.

As a workaround, I overload the OpenFOAM predictions over the graph from the paper:
![ventricle_results](https://github.com/user-attachments/assets/bb871c5c-1bc3-4532-9b7e-3b393a5d4a6c)

This looks OK; I think we can use this for the paper.

**Comment by iBatistic** [comment]
Yes, I see it now too. It's frustrating that they didn’t include either the script or the results from the diagram.

What you did looks great, I like it.
